venvUtils/ensureVenv.py

```diff
@@ -89,12 +79,32 @@ def createVenvAndPopulate():
 	shutil.copy(requirements_path, venv_orig_requirements_path)
 
 
+def createVenvAndPopulate():

```

Comment:
Might as well split these up for future modularity.

```suggestion
def createVenv():
```